ui/DesignSystem/Component/SourceBrowser/ClickOutside.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import { openDropdown, currentlyOpen } from "../../../src/dropdown";
+
+  let container;
+  export let shouldOpen = false;
```

Comment:
ideally components will use this as a wrapper & expand/close based on `shouldOpen`